modules/core/client/app/config.js

```diff
@@ -1,23 +1,29 @@
-'use strict';
+(function (window) {
```

Comment:
I disagree, as it's not part of the style guidelines.  I can remove window if you wish.